cadctl/cmd/remediate/remediate.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
I think we can start with just the cluster-missing command for now even

Reply:
Will move it up and change the commits